convert2rhel/actions/pre_ponr_changes/subscription.py

```diff
@@ -174,6 +174,7 @@ def run(self):
                 )
                 return
 
+            logger.task("Convert: Subscription Manager - Reload configuration")
```

Comment:
This addition kind of creeped in. I added this after spotting that the sub-man refresh output was hidden under a different TASK.

Reply:
+1